ui/lib/kubernetes/addon/components/page/role/details.hbs

```diff
@@ -0,0 +1,72 @@
+<PageHeader as |p|>
+  <p.top>
+    <nav class="breadcrumb" aria-label="breadcrumbs" data-test-breadcrumbs>
+      <ul>
+        <li data-test-crumb="overview">
+          <span class="sep">/</span>
+          <LinkTo @route="overview">{{@model.backend}}</LinkTo>
+        </li>
+        <li data-test-crumb="roles">
+          <span class="sep">/</span>
+          <LinkTo @route="roles">roles</LinkTo>
+        </li>
+        <li>
+          <span class="sep">/</span>
+          <span data-test-crumb="role">{{@model.name}}</span>
+        </li>
+      </ul>
+    </nav>
+  </p.top>
+  <p.levelLeft>
+    <h1 class="title is-3" data-test-header-title>
+      {{@model.name}}
+    </h1>
+  </p.levelLeft>
+</PageHeader>
+
+<Toolbar>
+  <ToolbarActions>
+    <ConfirmAction @buttonClasses="toolbar-link" @onConfirmAction={{this.delete}} data-test-delete>
+      Delete role
+    </ConfirmAction>
```

Comment:
small nit - this should have a separator here `<div class="toolbar-separator"></div>` - I noticed it missing from the k8 designs, but this has come up before and I think it just gets forgotten sometimes. Last I checked with Ivana it should always be there unless Delete is the only action

Reply:
I'm glad you pointed that out! I was looking at other examples in the code and noticed the separator but decided to go with the designs instead. I'll add it in and keep that in mind for the future!